server/handlers/default.go

```diff
@@ -35,6 +37,76 @@ func MainHandler(ctx context.Context, w http.ResponseWriter, r *http.Request) *e
 	return nil
 }
 
+// AtomicUpdateHandler will accept multiple TUF files and ensure that the storage
+// backend is atomically updated with all the new records.
+func AtomicUpdateHandler(ctx context.Context, w http.ResponseWriter, r *http.Request) *errors.HTTPError {
+	defer r.Body.Close()
+	s := ctx.Value("metaStore")
+	if s == nil {
+		return &errors.HTTPError{
+			HTTPStatus: http.StatusInternalServerError,
+			Code:       9999,
+			Err:        fmt.Errorf("Version store is nil"),
+		}
+	}
+	store, ok := s.(storage.MetaStore)
+	if !ok {
+		return &errors.HTTPError{
+			HTTPStatus: http.StatusInternalServerError,
+			Code:       9999,
+			Err:        fmt.Errorf("Version store not configured"),
+		}
+	}
+	vars := mux.Vars(r)
+	gun := vars["imageName"]
```

Comment:
Hmmm, might be a good idea. Should probably do that for the single file update too if it's a root.json being updated. We can add that check in both places for polish after freeze.